src/index.tsx

```diff
@@ -195,21 +195,28 @@ export default function AxeMode({ children, disabled }: AxeModeProps) {
     }
 
     if (childrenRef.current) {
-      if (idleId) {
+      if (idleId && 'cancelIdleCallback' in window) {
         // requestIdleCallback has no types:
         // https://github.com/microsoft/TypeScript/issues/21309
         // @ts-ignore
         cancelIdleCallback(idleId);
         setIdleId(null);
```

Comment:
I'd probably opt for a ref here vs. state. No reason to re-render just to capture the ID. Same comment applies elsewhere. 

`idleId.current = null`

Reply:
NOTE: I went ahead and pushed up some of these changes, but feel free to review and push back and/or revert if you think I missed the intent here.

Reply:
Thanks for that! Makes sense to me.